lumigator/python/mzai/schemas/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Are these schemas used in both API and SDK?
```

Comment:
not used for DB storage, those are in the repositories.  https://github.com/mozilla-ai/lumigator/blob/main/lumigator/python/mzai/backend/repositories/experiments.py

Reply:
Ok, thanks. Would it make sense to merge modules `repositories` and `records` since they're both ORM?